srcpkgs/libibmad/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'libibmad'
+pkgname=libibmad
+version=1.3.12
+revision=1
+wrksrc="OFED-3.18-2/SRPMS"
+build_wrksrc="libibmad-1.3.12"
+build_style=gnu-configure
+hostmakedepends="rpm cpio"
+makedepends="rdma-core-devel"
+depends="rdma-core"
```

Comment:
Not necessary because xbps-src detects the rdma-core dependency automatically:

```
=> libibmad-1.3.13_1: running pre-pkg hook: 04-generate-runtime-deps ...
   SONAME: libibumad.so.3 <-> rdma-core>=22.1_1
   SONAME: libc.so.6 <-> glibc>=2.29_1
```

Reply:
Thanks for pointing this out.  Corrected.